spec/yaml/swiftnav/sbp/tracking.yaml

```diff
@@ -38,7 +38,7 @@ definitions:
             type: u32
             desc: |
               Signal identifier being tracked - values 0x00 through 0x1F
-              represent GPS PRNs 1 through 32 respectively (PRN-1 notation);
+              represent GPS PRNs 1 through 32 respectively (PRN-minus-1 notation);
```

Comment:
Absolutely. Good catch. I'd flagged those previously but just hadn't made the changes in my local repo.

I'll do that now and the changes to the existing PR.